sklearn/datasets/openml.py

```diff
@@ -582,8 +569,11 @@ def fetch_openml(name=None, version='active', data_id=None, data_home=None,
         below for more information about the `data` and `target` objects.
 
     as_frame : boolean, default=False
-        If True, returns a Bunch where the data attribute is a pandas
-        DataFrame.
+        If True, where the data is a pandas DataFrame including columns with
+        appropriate dtypes (numeric, string or categorical). The target is
+        a pandas DataFrame or Series depending on the number of target_columns.
+        If ``return_X_y`` is True, then ``(data, target)`` will be pandas
```

Comment:
I am not sure this is necessary if the previous statement does not mention a Bunch. You could just say "this applies regardless of return_X_y"